docs/standard/serverless-architecture/index.md

```diff
@@ -0,0 +1,122 @@
+
+Some examples depicted herein are provided for illustration only and are fictitious. No real association or connection is intended or should be inferred.
+
+Microsoft and the trademarks listed at http://www.microsoft.com on the “Trademarks” webpage are trademarks of the Microsoft group of companies.
+
+Mac and macOS are trademarks of Apple Inc.
+
+The Docker whale logo is a registered trademark of Docker, Inc. Used by permission.
+
+All other marks and logos are property of their respective owners.
+
+Co-Authors:
+
+> **Jeremy Likness**, Sr. Cloud Developer Advocate, APEX, Microsoft Corp.
+
+Editors:
+
+Participants and reviewers:
+
+## Introduction
+
+Serverless is the evolution of cloud platforms in the direction of pure cloud native code that brings developers closer to business logic while insulating them from infrastructure concerns. It’s a pattern that doesn’t imply “no server” but rather, “less server”. Serverless code is event-driven. Code may be triggered by anything from a traditional HTTP web request to a timer or the result of uploading a file. The infrastructure behind serverless allows for instant scale to meet elastic demands and offers micro-billing to truly “pay for what you use.” Serverless requires a new way thinking and approach to building applications and is not the right solution for every problem. As a developer, you must decide:
+
+* What are the pros and cons of serverless?
+* Why should you consider serverless for your own applications?
+* How can you build, test, deploy, and maintain your serverless code? 
+* Where does it make sense to migrate code to serverless in existing applications, and what is the best way to accomplish this?
+
+## About this guide
+
+This guide focuses on cloud native development of applications that use serverless. It highlights the benefits and exposes the potential drawbacks of developing serverless apps and provides a survey of serverless architectures. It illustrates numerous examples of how serverless can be used and details various serverless design patterns.
+
+This guide explains the components of the Azure serverless platform and focuses specifically on implementation of serverless using Azure Functions. You will learn about triggers and bindings as well as how to implement serverless apps that rely on state using durable functions. Finally, business examples and case studies will help provide context and a frame of reference to determine whether serverless is the right approach for your projects.
+
+## Evolution of cloud platforms
+
+Serverless is the culmination of several iterations of cloud platforms.
+
+![Evolution from on-premises to serverless](./media/image1.png)
+
+Before the cloud a discernable boundary existed between development and operations. Deploying an application meant answering myriad questions like:
+
+* What hardware should I install?
```

Comment:
These lists should be 2nd person, not 1st.

We use 2nd person generally through our articles. You've followed that in most places other than these two lists of questions.